src/main/java/seedu/address/ui/ViewWindow.java

```diff
@@ -19,6 +19,7 @@ public class ViewWindow extends UiPart<Region> {
     private static final String FXML = "ViewWindow.fxml";
 
     public final Person person;
+    private final int maxWeeks = 13;
```

Comment:
Would it be better if this was referenced from `ParserUtil.MAX_WEEK` instead?

Reply:
Yea also I think a typical semester would have 14 weeks if we are going to count week 0?

Reply:
Yeah I agree with @DesSnowy, I will change that accordingly. And I put 13 weeks because we usually do not have classes or tutorials in week 0. I can still change it to include week 0 too if everyone is okay with it!

Reply:
Hmm I would feel that including it would at least be more appropriate since one might be confused whether week 1 means week 0 or week 1. What do others think about it?

Reply:
I think we can leave it as it is. In the developer/user guide we can make it such that future improvement will allow user to specify the total number of weeks they want.